File: IntegrationExamples/ObjCAppWithSwiftPackageManager/ObjCAppWithSwiftPackageManager/ViewController.m
//
//  ViewController.m
//  ObjCAppWithCocoaPods
//
//  Copyright © 2022 Radar Labs, Inc. All rights reserved.
//

#import "ViewController.h"
#import <Radar.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    NSBundle *radarBundle = [NSBundle bundleForClass:Radar.class];
    NSString *versionString = [radarBundle infoDictionary][@"CFBundleShortVersionString"];
    self.versionNumberLabel.text = versionString;
}


@end

File: IntegrationExamples/SwiftAppWithCarthage/SwiftAppWithCarthage/ViewController.swift
//
//  ViewController.swift
//  SwiftAppWithCocoaPods
//
//  Copyright © 2022 Radar Labs, Inc. All rights reserved.
//

import RadarSDK
import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var versionNumberLabel: UILabel!

    override func viewDidLoad() {
        super.viewDidLoad()

        let radarBundle = Bundle(for: Radar.self)
        let versionString = radarBundle.infoDictionary!["CFBundleShortVersionString"] as? String
        versionNumberLabel.text = versionString
    }

}
